lib/controllers/v1/observations_controller.js

```diff
@@ -322,6 +322,12 @@ ObservationsController.reqToElasticQueryComponents = function( req, callback ) {
       "taxon.ancestor_ids", params.taxon_id || params.taxon_ids ) );
   }
 
+  if( params.without_taxon_id || params.without_taxon_ids ) {
```

Comment:
I vote to only use `params.without_taxon_id` here. Looks like I made the API accept `taxon_ids` and `project_ids`, I suspect an early attempt to match the Rails API. Most other params like `user_id` or `place_id` accept multiple values w/o a corresponding `*_ids` param